core/lib/rom/struct.rb

```diff
@@ -43,11 +46,11 @@ module ROM
   #
   #   # see struct's schema attributes
   #
-  #   # model.schema.key(:id)
-  #   # => #<Dry::Types[id: Nominal<Integer meta={primary_key: true, source: :users}>]>
+  #   model.schema.key(:id)
+  #   # => #<Dry::Types[id: Nominal<Integer meta={primary_key: true, alias: nil, source: :users}>]>
```

Comment:
I think omitting insignificant meta keys is fine here

Reply:
Gotcha, just removed `alias`